app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -54,6 +59,20 @@ open class LockboxApplication : Application() {
         injectContext()
         setupLifecycleListener()
         setupSentry()
+
+        // Adjust Integration
+        val appToken = Constant.App.appToken
+
+        // ONLY USE SANDBOX when we are developing locally or testing.
+        // ONLY USE PRODUCTION when it is a real play store build.
+        val environment = AdjustConfig.ENVIRONMENT_SANDBOX
```

Comment:
Let's have this switch on the build variant rather than needing to do it manually here... I'm pretty sure this kind of thing is what the `isDebug()` helper method is for.